taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/HttpHeaderProvider.java

```diff
@@ -43,8 +43,23 @@ public HttpHeaders outboxRestApiHeaders() {
     }
   }
 
-  HttpEntity<String> prepareEntityFromBodyForCamundaRestApi(String requestBody) {
+  public HttpEntity<Void> prepareEntityFromBodyForCamundaRestApi() {
```

Comment:
The "FromBody" part of the method name doesn't make sense to me.  The request body isn't used in this method anymore. What do you think about renaming this method? For example "newEntityWithCamundaRestApiHeaders"?

Reply:
good point. I will change it.

Historically only these Methods exists

```Java

  HttpEntity<String> prepareEntityFromBodyForCamundaRestApi(String requestBody) {
    HttpHeaders headers = getHttpHeadersForCamundaRestApi();
    return new HttpEntity<>(requestBody, headers);
  }

  HttpHeaders getHttpHeadersForCamundaRestApi() {
    HttpHeaders headers = camundaRestApiHeaders();
    headers.setContentType(MediaType.APPLICATION_JSON);
    return headers;
  }

  HttpHeaders getHttpHeadersForOutboxRestApi() {
    HttpHeaders headers = outboxRestApiHeaders();
    headers.setContentType(MediaType.APPLICATION_JSON);
    return headers;
  }
```

but theses are not enough anymore. Thats why I created new methods without thinking about the names